doc/CRD-generator.md

```diff
@@ -4,13 +4,15 @@ The [CRD Generator annotation processing tool (APT)](../crd-generator/apt/README
 
 As a replacement, we're currently providing a new version of the API in `io.fabric8:crd-generator-api-v2` and a few tools to be able to leverage it in your projects.
 
+A migration guide can be found [here](CRD-generator-migration-v2.md). 
+
 The following list contains the available tooling:
 - [CRD Generator Maven Plugin](../crd-generator/maven-plugin/README.md): A Maven plugin that generates CRDs during the build process.
 - [CRD Generator CLI tool](../crd-generator/cli/README.md): A CLI tool that generates CRDs when executed.
 
 ## Quick start
 
-Add the CRD Generator plugin to your project or use the CLI tool:
+Add the CRD Generator plugin to your project:
```

Comment:
We need to provide a way for users who's projects aren't Maven-based to be able to use the generator.

I'd defer the removal of the CLI mention to when the Gradle plugin is available, and I'd even add an example of how to use it.
Otherwise I'm sure we'll get new issues and questions asking how to deal with this on non-Maven based projects (Gradle, SBT, etc.)

Reply:
Reverted the removal. You are right this is at the moment a big gap.